DKBeatMaker/DKBeatMaker/Conductor.swift

```diff
@@ -1,25 +1,25 @@
 import AudioKit
 
 class Conductor {
-    let midi = AKMIDI()
+    let midi: AKMIDI = AKMIDI()
```

Comment:
purely because I dont know :) why do we do this? is that to ensure the type of the property?

What would happen if you dont do that?